ang/civicase/DashboardTab.html

```diff
@@ -1,5 +1,194 @@
 <div class="civicase__dashboard__tab">
-  <div class="civicase__dashboard__tab__top"></div>
+  <div class="civicase__dashboard__tab__top">
+    <div class="panel panel-secondary civicase__dashboard__overview">
+      <div class="panel-body">
+        <div class="civicase__dashboard__overview-container">
+          <div class="civicase__dashboard__overview__title">
```

Comment:
The class does not describe the section, something like `.civicase__case-overview__flow` maybe would be more appropriate